dist/scss/wds-mixins/_community-header-theming.scss

```diff
@@ -0,0 +1,135 @@
+@import '../wds-functions/index';
+
+@mixin community-header-theming(
+	$header-background-color,
+	$color-text,
+	$color-links,
+	$color-page,
+	$color-page-border
+) {
+
+	// the same logic as $is-dark-wiki in oasis
+	$is-dark-theme: lightness($color-page) < 50;
+
+	$text-color: $wds-color-white;
+	@if luminance($header-background-color) > 0.3 {
+		$text-color: $wds-color-black;
+	}
+	$text-hover-color: rgba($text-color, 0.8);
+	$text-hover-transition: color .2s;
+
+	background-color: $header-background-color;
+
+	&::before {
+		background: linear-gradient(to right,rgba($header-background-color,0.5),rgba($header-background-color,0.5)),linear-gradient(to left,rgba(0,108,176,0) 200px,$header-background-color 430px);
+	}
+
+	.wds-button.wds-is-secondary {
+		background-color: rgba($header-background-color, 0.2);
+		border-color: $text-color;
+		color: $text-color;
+		transition-property: border-color,background-color,color;
+		transition-duration: .2s;
+
+		a {
+			color: $text-color;
+		}
+
+		svg {
+			fill: $text-color;
+		}
+	}
+
+	.wds-button.wds-is-secondary:hover {
+		background-color: rgba($header-background-color, 0.4);
+		border-color: $text-hover-color;
+		color: $text-hover-color;
+
+		svg {
+			fill: $text-hover-color;
+			transition: $text-hover-transition;
```

Comment:
transition should be set in a base selector (not :hover one), because when it's set as now, it will work only one-way. And I'm not sure if it is needed at all, I guess it will be inherited from parent.